packages/twenty-front/src/modules/object-record/relation-picker/components/SingleEntitySelectMenuItems.tsx

```diff
@@ -49,88 +53,163 @@ export const SingleEntitySelectMenuItems = ({
   isAllEntitySelected,
   isAllEntitySelectShown,
   onAllEntitySelected,
+  hotkeyScope = RelationPickerHotkeyScope.RelationPicker,
 }: SingleEntitySelectMenuItemsProps) => {
   const containerRef = useRef<HTMLDivElement>(null);
 
-  const entitiesInDropdown = [selectedEntity, ...entitiesToSelect].filter(
+  const createNewRecord = showCreateButton
+    ? {
+        __typename: '',
+        id: 'add-new',
+        name: 'Add New',
+      }
+    : null;
+
+  const selectNone = emptyLabel
+    ? {
+        __typename: '',
+        id: 'select-none',
+        name: emptyLabel,
+      }
+    : null;
+
+  const selectAll = isAllEntitySelectShown
+    ? {
+        __typename: '',
+        id: 'select-all',
+        name: selectAllLabel,
+      }
+    : null;
+
+  const entitiesInDropdown = [
+    selectAll,
+    selectNone,
+    selectedEntity,
+    ...entitiesToSelect,
+    createNewRecord,
+  ].filter(
```

Comment:
Making the `Add New`, `Select All`, `Select none` buttons as a part of dropdown entities since the `SelectableList` keys operate on ids. Open to suggestions if we can do it better :)

Reply:
It's not the prettiest but that's an acceptable hack imo!